vocab/constants/VOCAB_QUDT-CONSTANTS-v2.1.ttl

```diff
@@ -2569,6 +2570,7 @@ constant:ReducedPlanckConstant
   a qudt:PhysicalConstant ;
   dcterms:description "The \"Reduced Planck Constant}, or Dirac Constant, is used in applications where frequency is expressed in terms of radians per second (\\emph{angular frequency\") instead of cycles per second. In such cases a factor of \\(2\\pi\\) is absorbed into the constant."^^rdf:HTML ;
   qudt:applicableUnit unit:J-SEC ;
+  qudt:hasQuantityKind quantitykind:Action ;
```

Comment:
Similar LaTeX errors in description.

Change to:

dcterms:description "The \\(\\textit{Reduced Planck Constant}\\), or \\(\\textit{Dirac Constant}\\), is used in applications where frequency is expressed in terms of radians per second (\\(\\textit{angular frequency}\\)) instead of cycles per second. In such cases a factor of \\(\\(2\\pi\\) is absorbed into the constant."^^qudt:LatexString ;

Reply:
Ralph, a couple of questions:
1. Does this look correct, or should it have something about \textit:
```
qudt:LatexString
  rdf:type rdfs:Datatype ;
  rdfs:comment "A type of string in which some characters may be wrapped with '\\(' and '\\) characters for LaTeX rendering." ;
  rdfs:isDefinedBy <http://qudt.org/2.1/schema/qudt> ;
  rdfs:label "Latex String" ;
  rdfs:subClassOf xsd:string ;
.
```

2. Something isn't right here, and the description doesn't display at all in the normal EDG editor pane:
![image](https://user-images.githubusercontent.com/1130189/85463341-61760f80-b55b-11ea-9bec-aab75b02da4a.png)

Reply:
Agree that we defer the fix

Reply:
I'd like to do the merge then look at this afterwards.